DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,21 @@
    "metadata": {},
```

Comment:
<p><s>Looks like this equation isn't displaying properly?</s></p><p> </p><p>-&gt; Actually, this is just a problem here in "ReviewNB". Looks like all is good in the actual notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>